requirements_dev.txt

```diff
@@ -1,14 +1,15 @@
 PyYAML
-pip>=18.0.0
-wheel>=0.29.0
-watchdog>=0.8.3
-flake8>=2.6.0
-coverage>=4.1
 Sphinx
-sphinx_rtd_theme>=0.4.3
-pytest>=3.8
-pytest-runner>=4.2
 astroid
-pylint
 autoapi
+black
+coverage>=4.1
+flake8>=2.6.0
+pip>=18.0.0
+pylint
+pytest-runner>=4.2
+pytest>=3.8
 rstcheck
+sphinx_rtd_theme>=0.4.3
+watchdog>=0.8.3
+wheel>=0.29.0
```

Comment:
Where is watchdog applied? I only find it in requirements_dev

Reply:
Good catch, it is not any longer planned to be used..